beta/src/pages/_document.tsx

```diff
@@ -4,13 +4,15 @@
 
 import * as React from 'react';
 import {Html, Head, Main, NextScript} from 'next/document';
+import {getSandpackCssText} from '@codesandbox/sandpack-react';
 
 const MyDocument = () => {
   //  @todo specify language in HTML?
   return (
     <Html lang="en">
       <Head />
       <body className="font-sans antialiased text-lg bg-wash dark:bg-wash-dark text-secondary dark:text-secondary-dark leading-base">
+        <style id="sandpack">{getSandpackCssText()}</style>
```

Comment:
This will inject the CSS variables, and later the components styles after Suspense rendering Sandpack components